### ISSUE TITLE
Image cropping is not taken into account

### ISSUE THREAD

**Issue by joerou** [mwilliamson/mammoth.js#79]
Currently this library does a great job of getting the images from the docx file into the html output. However, if the image was cropped in word this cropping is not taken into account. Would it be possible to keep the fixes applied in word to the image when displaying in html?

**Comment by mwilliamson** [comment]
Thanks for the kind words. I suspect the two options are:

* Emit some inline styles to crop the image
* Generate a new, cropped image and reference that normally

The second seems vastly preferable, but I'm not sure when I'd get the time to take a look, and is non-trivial since Mammoth doesn't currently do any sort of image manipulation. Adding in something to do file manipulation might also make the browser JavaScript even bigger, and it's not exactly small at the moment, but that's just a guess. Feel free to leave a minimal example document so that if I do get a chance, I can make sure I'm actually trying to solve the right problem.

**Comment by joerou** [comment]
Of course, here is a word document with a cropped image of a strawberry. when you use mammoth to convert to html you will notice that the image contains more.

[Strawberry.docx](https://github.com/mwilliamson/mammoth.js/files/236716/Strawberry.docx)

Thanks again for taking a look into it!

**Comment by rvanlaak** [comment]
Would be great if cropping images, as it actually can be seen as a bug from a naggy consumer perspective. Will it require an extra dependency like https://www.npmjs.com/package/image-manipulation ?

**Comment by mwilliamson** [comment]
Thanks for the suggestion. It looks like that package is specifically for manipulation in the browser, so is unlikely to be suitable.
